skll/data/featureset.py

```diff
@@ -23,7 +23,8 @@ class FeatureSet(object):
 
     """
     Encapsulation of all of the features, values, and metadata about a given
-    set of data.
+    set of data. Note that FeatureSets can only be equal if the order of
```

Comment:
Nitpick:  Should probably use sphinx syntax for this:

```py
.. note::
    FeatureSets can only be equal if the order of the instances is identical 
    because these are stored as lists/arrays.
```

Might even be worth doing `.. warning`.